fusor-ember-cli/.gitignore

```diff
@@ -5,8 +5,9 @@
 /tmp
 
 # dependencies
-/node_modules
-/bower_components
+/node_modules*
```

Comment:
adding `*` allows us to have multiple dirs like `node_modules_1_13` and `node_modules_2_3` that can symlink from `node_modules`